pyproject.toml

```diff
@@ -33,6 +33,7 @@ alembic = "^1.13.2"
 psycopg2-binary = "^2.9.9"
 apprise = "^1.8.1"
 subliminal = "^2.2.1"
+websocket-client = "^1.8.0"
```

Comment:
Whats this?

Reply:
ah shit, I was supposed to remove that lol. good catch!

Reply:
fixed. updated deps as well.